src/Microsoft.TestPlatform.Client/Resources/xlf/Resources.de.xlf

```diff
@@ -39,9 +39,9 @@
         <note from="bb-metadata">fuzzyMatch="15" wordcount="14" adjWordcount="11.9" curWordcount="11.9"</note>
       </trans-unit>
       <trans-unit id="TestSessionTimeoutMessage">
-        <source>Aborting test run: test run timeout of {0} milliseconds exceeded.</source>
-        <target state="new">Canceling test run as it crossed testSessionTimeout: {0} milliseconds.</target>
-        <note></note>
+        <source>Canceling test run: test run timeout of {0} milliseconds exceeded.</source>
```

Comment:
this should be `Canceling test run: test run timeout of {0} milliseconds exceeded`.